resources/kubeapi-proxy.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "/kubeapi-proxy",
```

Comment:
@rimusz I got `kubeapi-proxy` as well as `marathon-lb`. Could we stick to a single one or have that as as an option of the Makefile target?

Reply:
we need both, as `kube-proxy` is just a dummy marathon app with traffic forward to k8s API

Reply:
Makes sense.